### ISSUE TITLE
Network stops working after adding iSCSI disk

### ISSUE THREAD

**Issue by arg** [churchers/vm-bhyve#396]
I noticed this weird behavior of the latest vm-bhyve on FreeBSD 12.2.

Here is my Ubuntu 20.04 VM configuration:

```
loader="uefi"
cpu=4
memory=8G
network0_type="virtio-net"
network0_switch="public"
disk0_type="virtio-blk"
disk0_name="disk0.img"
disk1_type="virtio-blk"
disk1_dev="iscsi"
disk1_name="iqn.iscsi.target.name"
graphics="no"
uuid="2968a825-4892-11ab-9411-004e01af2293"
network0_mac="aa:bb:cc:dd:ee:ff"
passthru0="0/2/0=2:0"
```
When I run this VM with `disk1` configured (it's already connected by `iscsictl`) I'm not able to reach my VM via SSH or ping it. Also if I connect to the VM via serial console, I can't reach any network (iSCSI drive is attached though), so the network connectivity looks to be completely off. Firewall is not enabled, setting network interface options from https://github.com/churchers/vm-bhyve/pull/395 didn't help.

**Comment by WanpengQian** [comment]
maybe the reason is mentioned in #357 
after adding iscsi, network slot changed.

**Comment by arg** [comment]
> maybe the reason is mentioned in #357
after adding iscsi, network slot changed.

Yes, I think so. I added a second disk (not even iSCSI, just a regular image) and my guest OS's network configuration got changed. I guess this ticket can be closed as a duplicate of #357.
